### PR TITLE
Use correct template name -  KendUI.Blank

### PR DESCRIPTION
**Pull request by nadyaA**
http://teampulse.telerik.com/view#item/307010

### CODE REVIEW

File: test/project.ts [hunk 1]
```diff
@@ -409,6 +409,17 @@ describe("project integration tests", () => {
 				});
 			});
 
+			it("When KendoUI.Blank is specified use KendoUI.Empty", () => {
+				options.template = "KendoUI.Blank";
+				project.createNewProject(projectName, projectConstants.TARGET_FRAMEWORK_IDENTIFIERS.Cordova).wait();
+				let projectDir = project.getProjectDir().wait();
+
+				_.forEach(mobileHelper.platformNames, platform => {
+					let cordovaFile = util.format("cordova.%s.js", platform).toLowerCase();
```
Comment (teobugslayer) [inline test/project.ts]: You may use backquoted string here.

File: test/project.ts [hunk 2]
```diff
@@ -409,6 +409,17 @@ describe("project integration tests", () => {
...
+					assert.isTrue(fs.existsSync(path.join(projectDir, cordovaFile)), util.format("Cordova KendoUI.Blank template does not contain mandatory '%s' file. This file is required in init command. You should check if this is problem with the template or change init command to use another file.", cordovaFile));
```
Comment (teobugslayer) [inline test/project.ts]: You may use backquoted string here

**Comment by rosen-vladimirov** [comment]
:+1: after green build :)

File: test/project.ts [hunk 3]
```diff
@@ -405,7 +405,18 @@ describe("project integration tests", () => {
 
 				_.forEach(mobileHelper.platformNames, platform => {
 					let cordovaFile = util.format("cordova.%s.js", platform).toLowerCase();
-					assert.isTrue(fs.existsSync(path.join(projectDir, cordovaFile)), util.format("Cordova KendoUI.Drawer template does not contain mandatory '%s' file. This file is required in init command. You should check if this is problem with the template or change init command to use another file.", cordovaFile));
+					assert.isTrue(fs.existsSync(path.join(projectDir, cordovaFile)), `Cordova KendoUI.Drawer template does not contain mandatory '%s' file. This file is required in init command. You should check if this is problem with the template or change init command to use another file`);
```
Comment (teobugslayer) [inline test/project.ts]: The message contains %s but I do not see an inserted string to replace it. Is this intended?

**Comment by nadyaA** [comment]
ping @teobugslayer

**Comment by teobugslayer** [comment]
:+1:

**Pull request merged**